aria_data_loaders/aria_data_utils/aria_live_streamer.py

```diff
@@ -134,20 +148,73 @@ def disconnect(self):
         self.device.streaming_manager.streaming_client.unsubscribe()  # type: ignore
         self.device_client.disconnect(self.device)  # type: ignore
 
+
+    def _rectify_image(self, image: np.ndarray) -> np.ndarray:
+        """
+        Rectify fisheye image based upon camera calibration parameters
+        Ensure you have set self._src_calib_param & self._dst_calib_param
+
+        Args:
+            image (np.ndarray): Image to be rectified or undistorted
+
+        Returns:
+            np.ndarray: Rectified image
+        """
+        assert self._rgb_calib_params is not None and self._dst_calib_params is not None
+        print(self._dst_calib_params)
+        print(self._rgb_calib_params)
+        aria_rect_rgb = distort_by_calibration(image, self._dst_calib_params, self._rgb_calib_params
+        )  # type: ignore
+        return aria_rect_rgb
+
+    def _rotate_img(self, img: np.ndarray, num_of_rotation: int = 3) -> np.ndarray:
+        """
+        Rotate image in multiples of 90d degrees
+
+        Args:
+            img (np.ndarray): Image to be rotated
+            k (int, optional): Number of times to rotate by 90 degrees. Defaults to 3.
+
+        Returns:
+            np.ndarray: Rotated image
+        """
+        img = np.ascontiguousarray(
+            np.rot90(img, k=num_of_rotation)
+        )  # GOD KNOW WHY THIS IS NEEDED -> https://github.com/clovaai/CRAFT-pytorch/issues/84#issuecomment-574683857
+        return img
+
     def process_frame(
-        self, frame: dict, detect_qr: bool = True, detect_hand_object: bool = True
-    ) -> dict:
+        self, frame: dict, outputs:dict, detect_qr: bool = True, detect_hand_object: bool = True
+    ) -> Tuple[np.ndarray, dict]:
         """
         Process the frame
         """
-        output = {}
+
+        # Initialize camera_T_marker to None & object_scores to empty dict for current image frame
+        camera_T_marker = None
+        object_scores = {}
+
+        # Rectify current image frame
+        frame["image"] = self._rectify_image(image=frame.get("image"))
```

Comment:
take a look at the new code in aria_live_streamer. frame has "raw_image" which is only rectified and a "image" which is the rect+rot one.

Could you update your code to play friendly with that?